README.md

```diff
@@ -28,3 +28,10 @@ The API uses mostly Laravel apiResource endpoints. All of the exposed endpoints
 | POST /wishlists/item | <ul><li>wishlist_id(int)</li><li>product_id(int)</li><li>qty(int)</li></ul> | Adds a new item to the given wishlist |
 | PATCH /wishlists/item/{id} | <ul><li>description(str, max 255)</li><li>qty(int)</li></ul> | Updates the data of an item |
 | DELETE /wishlists/item/{id} | None | Deletes an item |
+
+## Colors
+When you install Rapidez, the colors are already configured in your `tailwind.config.js`. This ensures that your colors are consistently applied across all elements.
+Alternatively, if you are using this package standalone, you can copy these variables and customize them to your liking. Colors can be found in the [Rapidez docs](https://docs.rapidez.io/3.x/theming.html#colors).
```

Comment:
Using this package "standalone" without Rapidez isn't really possible, I'm not sure why that part is in here. The composer.json requires `rapidez/core`, the code listens to the `vue:loaded` event which is Rapidez-specific, and we even use the `window.magentoCart` and `window.rapidezAPI` functionality to fetch data.

Reply:
Check I will remove it from the docs :)